### PR TITLE
ROX-12908: adding external entities/cidr block svg to network graph

### PR DESCRIPTION
**Pull request by alwayshooin**
should look like below: 
image

### CODE REVIEW

File: ui/apps/platform/src/Containers/NetworkGraph/types/topology.type.ts [hunk 1]
```diff
@@ -40,6 +40,13 @@ export type NamespaceData = {
 
 export type NetworkPolicyState = 'none' | 'both' | 'ingress' | 'egress';
 
+export type NodeDataType =
+    | 'DEPLOYMENT'
+    | 'EXTERNAL_GROUP'
+    | 'EXTERNAL_ENTITIES'
+    | 'CIDR_BLOCK'
+    | 'EXTRANEOUS';
+
```
Comment (alwayshooin) [inline ui/apps/platform/src/Containers/NetworkGraph/types/topology.type.ts:49]: not sure what the best way to have this exist along side the actual `type` strings in the specific data types of each kind of node below. open to suggestions!

File: ui/apps/platform/src/Containers/NetworkGraph/components/StyleNode.tsx [hunk 2]
```diff
@@ -56,22 +51,24 @@ type StyleNodeProps = {
...
 
-const getTypeIcon = (dataType?: DataTypes): any => {
-    switch (dataType) {
-        case DataTypes.Alternate:
-            return AlternateIcon;
+const getTypeIcon = (type?: NodeDataType): any => {
+    switch (type) {
+        case 'EXTERNAL_ENTITIES':
+        case 'CIDR_BLOCK':
+            return PficonNetworkRangeIcon;
```
Comment (sachaudh) [inline ui/apps/platform/src/Containers/NetworkGraph/components/StyleNode.tsx:58]: Such a strange name

File: ui/apps/platform/src/Containers/NetworkGraph/types/topology.type.ts (hunk 1 above)
Comment (sachaudh) [reply ui/apps/platform/src/Containers/NetworkGraph/types/topology.type.ts:49]: This is fine. I don't know if there is a fancy Typescript way of going through the `CustomNodeData` type and extracting the value for the `type` field similarly. Seems like overkill anyways

**Review by sachaudh — APPROVED** [review]

File: ui/apps/platform/src/Containers/NetworkGraph/types/topology.type.ts (hunk 1 above)
Comment (vjwilson) [reply ui/apps/platform/src/Containers/NetworkGraph/types/topology.type.ts:49]: Could you use the TS `as const` pattern where the actual constants are exported to get both "for the price of one"?
```suggestion
export const type = {
    DEPLOYMENT: 'DEPLOYMENT',
    EXTERNAL_GROUP: 'EXTERNAL_GROUP',
    EXTERNAL_ENTITIES: 'EXTERNAL_ENTITIES',
    CIDR_BLOCK: 'CIDR_BLOCK',
    EXTRANEOUS: 'EXTRANEOUS',
} as const;
```

Comment (alwayshooin) [reply ui/apps/platform/src/Containers/NetworkGraph/types/topology.type.ts:49]: thanks, will explore this in a later PR

**Pull request merged**